api/src/certification/evaluation/application/scenario-simulator-route.js

```diff
@@ -30,7 +30,7 @@ const register = async (server) => {
               capacity: Joi.number().min(-8).max(8).required(),
               accessibilityAdjustmentNeeded: Joi.boolean(),
               locale: Joi.string()
-                .valid(...Object.values(LOCALE))
+                .valid(...getChallengeLocales())
```

Comment:
Une remarque pour référence ultérieure mais sans impact apparent sur le fonctionnement (route utilisée uniquement pour de la simulation de déroulé de certification).

Il y a un petit distingo qui s'opère entre ce qui était anciennement renvoyé par la `...Object.values(LOCALE)` -> `['en', 'fr-fr', 'fr', 'nl', 'es']` et `...getChallengeLocales()` -> `['en', 'fr', 'fr-fr', 'nl', 'es', 'it', 'de']`

Reply:
Oui la liste `getChallengeLocales() -> ['en', 'fr', 'fr-fr', 'nl', 'es', 'it', 'de']` correspond à ce que nous avons trouvé en production actuellement.

Je ne marque pas cette conversation comme résolue pour que ces échanges restent visibles.